src/Services/SaveScan.php

```diff
@@ -78,7 +78,7 @@ protected function createOrUpdate($namespace, $group, $key, $mainKey=null): void
             $locals = config('filament-translations.locals');
             $text = [];
             foreach ($locals as $locale => $lang) {
-                $text[$locale] = Lang::get($mainKey,[],$locale);
+                $text[$locale] = $locale === 'en' ? Lang::get($mainKey, [], $locale) : '';
```

Comment:
I left the string empty because I actually want the users to see that the string is still empty and needs to be translated. Adding a string would suggest that it already has a translation.